src/content/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana.mdx

```diff
@@ -43,7 +43,7 @@ Follow these steps to add New Relic as a Prometheus data source for Grafana.
 8. Click **Save & Test**.
 
 <Callout variant="tip">
-  If your line graphs appear as a series of dots and not as connected lines, you can change in Graphana's **[Graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/graph-panel/)** and selecting **Stacking and null value > connected**.
+  If your line graphs appear as a series of dots and not as connected lines, you can change in Grafana's **[Graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/graph-panel/)** and selecting **Stacking and null value > connected**.
```

Comment:
The change itself looks fine, but the sentence it's contained in doesn't really read for me. Bolded part:

> If your line graphs appear as a series of dots and not as connected lines, **you can change in** Grafana's graph panel...

Reply:
Ah, weird, yeah should have looked at that closer. Did a second PR to fix.